content/general/concepts/cybersecurity/cybersecurity.md

```diff
@@ -20,7 +20,7 @@ Cybersecurity is the practice of protecting computer systems and networks from b
 - Educating a company's stakeholders on the importance of cyber and information security.
 - Planning in disaster recovery and responses to cybersecurity breaches.
 - Closing gaps in network security and mitigating vulnerabilities.
-- Defending against various [cyber-attacks](https://www.codecademy.com/resources/docs/general/cyber-attacks).
+- Defending against various [cyber-attack](https://www.codecademy.com/resources/docs/general/cyber-attack).
```

Comment:
Think you could leave the plural form; doesn't agree otherwise.
```suggestion
- Defending against various [cyber-attacks](https://www.codecademy.com/resources/docs/general/cyber-attack).
```